### PR TITLE
[DRAFT] Update instructions to post-commit policy web page

### PR DESCRIPTION
**Pull request by HuangLED**
A summary of discussions in dev mailing list: https://lists.apache.org/thread.html/a558b4df08be926b5f4ee268e205f084d7b2442da4b4d8cfc7bfe39f@%3Cdev.beam.apache.org%3E

Staged URL: http://apache-beam-website-pull-requests.storage.googleapis.com/7095/contribute/postcommits-policies-details/index.html#inform_community

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

File: website/src/contribute/postcommits-policies-details.md [hunk 1]
```diff
@@ -91,3 +91,21 @@ When you implement a fix for a post-commit test failure, add a new pre-commit
...
+the following three questions:
+
+1. Descriptions on what the issue is.
+2. Does a revert fix it? (Or it is supposed to be fixed differently)
+3. Is a revert the best way to fix it?
+
+It is encouraged to bring the discussion to the dev mailing list as well. 
+Ideally, after the incident, we prefer to have discussions regarding
+whether we should extend tests in Beam repository, with the goal of
+catching the similar issue early in the future.
```
Comment (mxm) [inline website/src/contribute/postcommits-policies-details.md]:
```suggestion
catching similar issues early in the future.
```

File: website/src/contribute/postcommits-policies-details.md [hunk 2]
```diff
@@ -91,3 +91,21 @@ When you implement a fix for a post-commit test failure, add a new pre-commit
 test that will detect similar failures in the future. For example, you can
 implement a new unit test that covers a problematic code branch.
 
+## Inform the community if Beam breaks downstream projects {#inform_community}
+
+There are multiple projects depend on Beam, and contain tests that are
```
Comment (mxm) [inline website/src/contribute/postcommits-policies-details.md]:
```suggestion
There are multiple external projects depending on Beam which contain tests that are
```

File: website/src/contribute/postcommits-policies-details.md [hunk 3]
```diff
@@ -91,3 +91,21 @@ When you implement a fix for a post-commit test failure, add a new pre-commit
...
+outside of Beam repository. For example, Dataflow, Samza runner, and IBM Streams. 
+
+When a depending project encounters an issue caused by (a PR) in beam,
```
Comment (mxm) [inline website/src/contribute/postcommits-policies-details.md]:
```suggestion
When an external project encounters an issue caused by (a PR) in Beam
```

File: website/src/contribute/postcommits-policies-details.md [hunk 4]
```diff
@@ -91,3 +91,21 @@ When you implement a fix for a post-commit test failure, add a new pre-commit
 test that will detect similar failures in the future. For example, you can
 implement a new unit test that covers a problematic code branch.
 
+## Inform the community if Beam breaks downstream projects {#inform_community}
+
+There are multiple projects depend on Beam, and contain tests that are
+outside of Beam repository. For example, Dataflow, Samza runner, and IBM Streams. 
+
+When a depending project encounters an issue caused by (a PR) in beam, 
+and in consequence, one requests for a change in the Beam repository
```
Comment (mxm) [inline website/src/contribute/postcommits-policies-details.md]:
```suggestion
and, in consequence, requests for a change in the Beam repository,
```

File: website/src/contribute/postcommits-policies-details.md [hunk 5]
```diff
@@ -91,3 +91,21 @@ When you implement a fix for a post-commit test failure, add a new pre-commit
...
 implement a new unit test that covers a problematic code branch.
 
+## Inform the community if Beam breaks downstream projects {#inform_community}
+
+There are multiple projects depend on Beam, and contain tests that are
+outside of Beam repository. For example, Dataflow, Samza runner, and IBM Streams. 
+
+When a depending project encounters an issue caused by (a PR) in beam, 
+and in consequence, one requests for a change in the Beam repository 
+accordingly, the first thing is to create a JIRA entry that addresses
```
Comment (mxm) [inline website/src/contribute/postcommits-policies-details.md]:
```suggestion
the first thing is to create a JIRA entry that addresses 
```

**Comment by HuangLED** [comment]
Thanks for the suggestions!  

All edits applied in github UI and then squashed into one commit.

**Comment by robertwb** [comment]
Thanks. Looks good to me too.

**Pull request merged**